### DISCUSSION TITLE
GitHub Community/community

### DISCUSSION THREAD

**Discussion by Ntshangase** [community/community#55613] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello,

There is a broken link on https://github.com/community/community, The `how to give good feedback` link is returning an `error 404`.
<br>
![Github Community bug](https://github.com/community/community/assets/62051440/81435596-53f8-4223-8562-9e8a679f7820)
![Github 404](https://github.com/community/community/assets/62051440/2a2d7ee9-041a-4a81-9a34-5f65ab4e7fe3)

**Comment by hardal7** [comment, marked as answer]
Hi Ntshangase,  The feedback guidance page has been moved to:
https://github.com/orgs/community/discussions/1
Hope this helps
